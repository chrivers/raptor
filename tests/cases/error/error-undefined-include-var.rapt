# line 1
INCLUDE dummy.rapt foo=bar
# line 3