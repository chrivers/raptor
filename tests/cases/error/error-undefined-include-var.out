error: Script Error
 --> error-undefined-include-var.rapt:1:24
  |
1 | INCLUDE dummy.rapt foo=bar
  |                        ^^^ bar
2 | 
  |
